raster_loader/io.py

```diff
@@ -413,11 +417,17 @@ def rasterio_windows_to_records(
         bounds_polygon = shapely.Polygon(shapely.wkt.loads(bounds_geog))
         center_coords = list(*bounds_polygon.centroid.coords)
         center_coords.append(resolution)
+        band_metadata = {
+            "name": band,
+            "type": str(band_type),
+            "nodata": raster_dataset.nodata,
+            "value_field": band_name,
```

Comment:
This field is not part of the spec, should be removed

Reply:
this is the refecence to the column where data are available, if we allow custom column name it is not possible to remove it.

Reply:
The name should be included in the "name" field, and this is the name of the column with the raster data